x-pack/plugin/sql/qa/server/src/main/resources/pivot.csv-spec

```diff
@@ -201,6 +201,51 @@ null                 |10044          |Mingsen        |F              |1994-05-21
 // end::sumWithoutSubquery
 ;
 
+sumWithInnerAggregateSumOfSquares
+schema::birth_date:ts|emp_no:i|first_name:s|gender:s|hire_date:ts|last_name:s|1:d|2:d
+SELECT * FROM test_emp PIVOT (SUM_OF_SQUARES(salary) FOR languages IN (1, 2)) LIMIT 5;
```

Comment:
This query fails for me in a REST test (using a REST client): it's only returning 3 rows, whereas it should have returned 5. We need to look into why the test passes and/or why the REST call with the same query fails. @palesz please, open an issue for this investigation.

Reply:
The JDBC integration tests fetch all the rows and follow the cursor (hence 5 rows shows up there). The tests are valid and successful, no issue there. Created https://github.com/elastic/elasticsearch/issues/65982 to investigate this unexpected behaviour related to `PIVOT+LIMIT`.